src/cli.rs

```diff
@@ -293,24 +325,70 @@ pub struct Cli {
 
     /// The request URL, preceded by an optional HTTP method.
     ///
-    /// METHOD can be `get`, `post`, `head`, `put`, `patch`, `delete` or `options`.
-    /// If omitted, either a GET or a POST will be done depending on whether the
-    /// request sends data.
+    /// If the method is omitted, it will default to either GET or POST
+    /// depending on whether the request contains a body or not.
+    ///
+    /// The URL scheme defaults to "http://" normally, or "https://" if
+    /// the program is invoked as "xhs".
+    ///
+    /// A leading colon works as shorthand for localhost. ":8000" is equivalent
+    /// to "localhost:8000", and ":/path" is equivalent to "localhost/path".
+    ///
+    /// URLs can have a leading "://" which allows quickly converting a URL
+    /// into a valid xh or HTTPie command. For example "http://httpbin.org/json"
+    /// becomes "http ://httpbin.org/json".
     #[clap(value_name = "[METHOD] URL")]
     raw_method_or_url: String,
 
     /// Optional key-value pairs to be included in the request
     ///
-    ///   - key==value to add a parameter to the URL
-    ///   - key=value to add a JSON field (--json) or form field (--form)
-    ///   - key:=value to add a complex JSON value (e.g. `numbers:=[1,2,3]`)
-    ///   - key@filename to upload a file from filename (with --form)
-    ///   - @filename to use a file as the request body
-    ///   - header:value to add a header
-    ///   - header: to unset a header
-    ///   - header; to add a header with an empty value
+    /// The separator is used to determine the type i.e. header, request body, query string, etc.
+    ///
+    /// Possible REQUEST_ITEM types are:
+    ///
+    ///     key==value
+    ///         Add a query string to the URL.
+    ///
+    ///     key=value
+    ///         Add a JSON property (--json) or form field (--form) to
+    ///         the request body.
+    ///
+    ///     key=@file
+    ///         Add a JSON property (--json) or form field (--form) from a
+    ///         file to the request body.
     ///
-    /// A backslash can be used to escape special characters (e.g. weird\:key=value).
+    ///     key:=value
+    ///         Add a field with literal JSON value to the request body,
+    ///         e.g. numbers:=[1,2,3] enabled:=true.
```

Comment:
I'm worried these examples don't stand out enough from the surrounding text. Maybe we can put quotes around them? Or use special roff formatting?

Reply:
I have now added double quotes to the examples inside `[REQUEST_ITEM ...]`.